### PR TITLE
Informations about Frontend Migration

### PR DESCRIPTION
**Pull request by lst97**
Add information migration progress.

_Any italic text should be deleted from the final Pull Request text, including this line_

# Description

_Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change._

Fixes # (issue)
## Type of change
Remove duplication file "Testing Decision Process"

- [x] Update & New Documentation

# How Has This Been Tested?
Not yet tested, do it later

## Testing Checklist:
pylint

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Review by lst97 — COMMENTED** [review]
Initial pull request
Waiting for video to be upload
Some link may need to change.

**Review by lst97 — COMMENTED** [review]
Fix indentation in syntax.coffee

**Review by lst97 — COMMENTED** [review]
commented for linelint check.

**Comment by lst97** [comment]
Hey @leocomsci I need your help to solve the Linelint error
[EOF Rule] File "docs/Company Operations/Company Documentation Team/# Password Guideline.md" has lint errors
[EOF Rule] File "docs/OnTrack/Front End Migration/Code References/CoffeeScript/syntax.coffee" has lint errors

**Comment by leocomsci** [comment]
HI @lst97 , there is a template that you can follow in order to generate a documentation. I attached this link below: 
https://github.com/thoth-tech/documentation/blob/main/docs/OnTrack/Documentation/OnTrack%20Documentation%20Template.md

**Comment by leocomsci** [comment]
Don't forget to add @ShaeChristmas  as a reviewer

File: docs/OnTrack/Front End Migration/Code References/CoffeeScript/syntax.coffee [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# # 1. Variables and Scope
```
Comment (leocomsci) [inline docs/OnTrack/Front End Migration/Code References/CoffeeScript/syntax.coffee:1]: Those documentation are same as what we did in our team channel. We already asked Nelson to add the Documentation Template to make this documentation make more sense. This is good to me. @ShaeChristmas @Ishrat420 Is there anything that I and Nelson should consider or fix? 
- Moreover, It seems that there is an error relating to linelint in check section of this PR. I have no idea about that error. Let me know if you have any ideas

**Review by ShaeChristmas — APPROVED** [review]
Everything looked good to me, just make sure that the md files pass the prettier and vale checks.

**Review by ShaeChristmas — CHANGES_REQUESTED** [review]
Please ensure that Vale and Prettier checks succeed

**Comment by lst97** [comment]
@ShaeChristmas @Ishrat420 @leocomsci Check PASSED, please check.

**Pull request merged**